src/AutoRest.CSharp.V3/AutoRest/Plugins/MemberRemoverRewriter.cs

```diff
@@ -103,7 +105,7 @@ private bool ShouldRemove(ISymbol? symbol)
                     foreach (var member in members)
                     {
                         if (!member.Equals(symbol) &&
-                            member.DeclaringSyntaxReferences.Any())
+                            IsDeclaredInNonGeneratedCode(member))
```

Comment:
Don't remove members from generated files to avoid hiding the generator problems.